### PR TITLE
Change info colors for smbv1 and signing to bold for readability

### PR DESCRIPTION
**Pull request by NeffIsBack**
Bold, or not bold, that is the question

image

Also fixes issues like #92 when upgrading from an old config file

### CODE REVIEW

**Comment by mpgn** [comment]
image

**Comment by mpgn** [comment]
image

**Comment by mpgn** [comment]
image

File: cme/data/cme.conf [hunk 1]
```diff
@@ -2,10 +2,11 @@
 workspace = default
 last_used_db = smb
 pwn3d_label = Pwn3d!
-audit_mode =
+audit_mode = False
```
Comment (mpgn) [inline cme/data/cme.conf:5]: The value need to be empty by default

image

**Comment by mpgn** [comment]
image

File: cme/config.py [hunk 2]
```diff
@@ -1,8 +1,14 @@
 # coding=utf-8
 import os
+from os.path import join as path_join
 import configparser
-from cme.paths import CME_PATH
+from cme.paths import CME_PATH, DATA_PATH
 from cme.first_run import first_run_setup
+from cme.logger import cme_logger
+import ast
```
Comment (mpgn) [inline cme/config.py:8]: from ast import literal_eval

File: cme/data/cme.conf (hunk 1 above)
Comment (NeffIsBack) [reply cme/data/cme.conf:5]: oh i see haha

**Review by mpgn — APPROVED** [review]

**Pull request merged**